lblod/frontend-lokaal-mandatenbeheer
# Ben/lmb 410 fix bestuursperiod selector

## Description

In the bestuursperiod selector only the bestuursperiods of one bestuursorgaan where shown, now all bestuursperiods are shown.

## How to test

Create a new bestuursperiod on a bestuursorgaan and check that this bestuursperiod actually show up in a bestuursperiod-selector. This can be validated with the bestuurperiod selector in the fracties or in the organen route.
